viewer/packages/rendering/src/glsl/sector/primitives/eccentricCone.frag

```diff
@@ -26,15 +25,13 @@ in vec4 v_centerB;
 in float height;
 in vec3 v_color;
 in vec3 v_normal;
+
 in highp vec2  v_treeIndexPacked;
 
 void main()
 {
     highp float v_treeIndex = unpackTreeIndex(v_treeIndexPacked);
     NodeAppearance appearance = determineNodeAppearance(colorDataTexture, treeIndexTextureSize, v_treeIndex);
```

Comment:
I think we should somehow document that we are doing the same lookup in the vertex and fragment shaders because we are hitting the attribute limit. I suggest just adding a comment along the lines of `// Redo appearance texture lookup from vertex shader due to limit in transferable attributes`, in this file and all the other fragment shaders where this occurs.

Open to other suggestions on how to document this.

Reply:
Sounds like a good idea! Just pushed that